keyboards/handwired/jscotto/scottoslant/keymaps/default/keymap.c

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright 2022 Joe Scotto
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+// Tap Dance declarations
```

Comment:
Could the tap dance stuff be moved to another keymap? 

We want to keep the default keymap fairly simple, as it's cloned when making a new keymap.  For newer users, complicated keymaps tend to cause more issues than they solve. 

That said, adding a "stock" type keymap with more features is fine.

Reply:
Same as my other board... its for CMD+space mainly.

Reply:
https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs

- default (and via) keymaps should be “pristine”
  - bare minimum to be used as a “clean slate” for another user to develop their own user-specific keymap
  - standard layouts preferred in these keymaps, if possible
  - default keymap should not enable VIA – the VIA integration documentation requires a keymap called via

Having the tap dance stuff would be fine for a "stock" type keymap, but the default kept and simple and pristine as possible.  

There are a number of reasons for this, but the biggest one is that the default keymap is copied when creating new keymaps and more advanced features can (and do) trip up a lot of new users.